src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -127,9 +130,10 @@ class LocalBackend extends Backend with LazyLogging {
     val argv = Seq("/bin/bash", "-c", s"cat $commandFile | $dockerRun /bin/bash <&0")
     logger.debug("Executing call with argv: " + argv)
 
-    // The ! to the ProcessLogger captures standard output and error.
-    val rc: Int = argv ! ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline)
-    Vector(stdoutWriter, stderrWriter).foreach { _.flushAndClose() }
+    val process = argv.run(ProcessLogger(stdoutWriter writeWithNewline, stderrWriter writeWithNewline))
+    callAbortRegisteringFunction.registrationFunction(AbortFunction(() => process.destroy()))
+    val rc: Int = process.exitValue()
+    Vector(stdoutWriter, stderrWriter) foreach { _.flushAndClose() }
```

Comment:
Could you please add some comments here that the code unfortunately doesn't do what we hoped?  :smile: